homeassistant/components/fireservicerota/sensor.py

```diff
@@ -0,0 +1,154 @@
+"""Sensor platform for FireServiceRota integration."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import ATTRIBUTION, DOMAIN
```

Comment:
DOMAIN in a platform should only represent the platforms own domain (in this case sensor)
```suggestion
from .const import ATTRIBUTION, DOMAIN as FIRESERVICEROTA_DOMAIN
```